.github/workflows/check-catalyst.yaml

```diff
@@ -526,6 +526,11 @@ jobs:
     - name: Checkout Catalyst repo
       uses: actions/checkout@v4
 
+    - name: Install device dependencies (OpenQasm device)
+      run: |
+        pip install amazon-braket-pennylane-plugin
+        echo "AWS_DEFAULT_REGION=us-east-1" >> $GITHUB_ENV
```

Comment:
I think think we need to also set the default region as local tests are only tested,
```suggestion
```

Reply:
I think this is a change that should be external to this PR. Tagging @dime10 for visibility. I'll resolve this once noted.

Reply:
Resolving this for now.